File: Assignments/Assignment_01/Summer-22/Task_02.java
import java.util.Arrays;

public class Assignment_01 {
    public static void main(String[] args){
        int[] source_2 = {10,20,30,40,50,60};
        System.out.println(Arrays.toString(rotateLeft(source_2,3)));
    }
  

public static int[] rotateLeft(int[] source, int k){
        int[] newArray = new int[source.length];
        for (int i=0,tmp=0; i<(source.length); i++,k++){
            newArray[i] = source[k%source.length];
        }
        return newArray;
    }
  
}

File: Assignments/Assignment_01/Summer-22/Task_02.py
# Test 02: Rotate Left k cell
def rotate_left(source, k):
  # TO DO
  # Hint, You can write a function for left rotate once and then use it
  length = len(source)

  for j in range(k):   
    tmp = source[0]
    for i in range(len(source)-1):
        source[i] = source[i+1]

    source[len(source)-1] = tmp

  return source
